src/styles/_variables.scss

```diff
@@ -30,39 +30,56 @@
     --mynah-color-toggle-reverse: rgba(0, 0, 0, 0.5);
 
     --mynah-color-syntax-bg: var(--vscode-terminal-dropBackground);
-    --mynah-color-syntax-variable: var(
```

Comment:
These should remain as is, the theme you'll use for the highlight.js should use those css custom properties.
We're not gonna get rid of these custom properties. The reason we have those is to not make any change on the usage of the MynahUI even if we change the library we use, like in this case from prism.js to highlight.js.

Reply:
And by default, we're using VSCode theme colors, however you can add all new color properties as fallbacks:
For example:

```css
--mynah-color-syntax-variable: var(--vscode-gitDecoration-modifiedResourceForeground,
        var(--vscode-debugTokenExpression-name, #986801))
    )
```